src/test/java/pageRefactory/HomePage.java

```diff
@@ -0,0 +1,34 @@
+package pageRefactory;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage {
+    @FindBy(css = "img.avatar")
+    WebElement avatarIcon;
+    @FindBy(css = "#sidebar > #playlists :nth-child(6)")
+    WebElement aPlaylist;
+    @FindBy(css = "input[name='name']")
+    WebElement editPlaylistName;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```